pkg/rancher-desktop/components/UpdateStatus.vue

```diff
@@ -138,6 +158,10 @@ class UpdateStatus extends UpdateStatusProps {
     return this.applying ? 'Applying update...' : 'Restart Now';
   }
 
+  get unsupportedUpgradeAvailable() {
+    return !this.hasUpdate && this.updateState?.info?.unsupportedUpgradeAvailable;
```

Comment:
This is intentional. I didn't want the "unsupported upgrade avaliable" card to be shown at the same time as the main update card. I didn't want to confuse the user by simultaneously saying that an update is available, and that an unsupported update is available.

I suppose we *could* show both at the same time, though it would probably entail scrolling, which AFAIK we're trying to avoid.

Another option that comes to mind is to modify the main update card to handle the case where only an unsupported update is available.

It may be worth it to get a UI person to look at this.

Reply:
Ok, but won't `this.hasUpdate` also be true when the Upgrade is unsupported? In which case this expression would always be false.

(Now that I typed this: is `Update` and `Upgrade` used interchangeably here? Should we use one term consistently?)